deployment/tfhub/object-detection/base64/object-detection-base64-predict.ipynb

```diff
@@ -1,241 +1,293 @@
+        "This model identifies objects present in images, returning its scores, labels and bounding boxes.\n",
+        "\n",
+        "We are using a dataset from [UCF](https://www.crcv.ucf.edu/data/GMCP_Geolocalization/#Dataset) and the model [SSD Mobilenet V2](https://tfhub.dev/google/openimages_v4/ssd/mobilenet_v2/1) from [TensorFlow Hub](https://www.tensorflow.org/hub)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Wh9NwPMRIu5n"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/VertaAI/examples/blob/main/deployment/tfhub/object-detection/base64/object-detection-base64-predict.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Fm0ffqV_Iuw_"
+      },
+      "source": [
+        "## Dependencies"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qJSXlocsIuln"
+      },
+      "source": [
+        "This notebook has been tested with **Python 3.8.15** and the following package versions:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "YVkBoQBGzdd-"
+      },
+      "outputs": [],
+      "source": [
+        "%%capture\n",
+        "!pip install numpy==1.23.5\n",
+        "!pip install verta==0.21.1"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8aNS5l4PJ__e"
+      },
+      "source": [
+        "## Imports"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ASXSPO_ezkT-"
+      },
+      "outputs": [],
+      "source": [
+        "import base64\n",
+        "import concurrent.futures\n",
+        "import numpy as np\n",
+        "import os\n",
+        "import requests\n",
+        "import time\n",
+        "import json\n",
+        "\n",
+        "from verta import Client"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p-JtMXyhKK83"
+      },
+      "source": [
+        "## Verta Set Up"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "cXeEZ9yRKKXf"
+      },
+      "outputs": [],
+      "source": [
+        "os.environ['VERTA_HOST'] = ''\n",
+        "os.environ['VERTA_EMAIL'] = ''\n",
+        "os.environ['VERTA_DEV_KEY'] = ''"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "4eRAOufkKRw7",
+        "outputId": "64a6a46c-6827-4cbf-e96f-d279654276ee"
+      },
+      "outputs": [],
+      "source": [
+        "client = Client(os.environ['VERTA_HOST'] , debug=True)"
```

Comment:
I'd recommend not using `debug=True` in the final notebook, because it's a bit messy 😬 

```suggestion
        "client = Client(os.environ['VERTA_HOST'])"
```